### DISCUSSION TITLE
Project future

### DISCUSSION THREAD

**Discussion by unlobito** [unlobito/ha-hildebrandglow#55] [Announcements]
Hi folks,

I've been unfortunately quiet here over the last few months. I haven't had a working smart meter since the 30th of December (https://github.com/unlobito/ha-hildebrandglow/issues/46#issuecomment-1004863060) and my attempts to get a SMETS2 meter commissioned have been fruitless so far.

<details>
  <summary>What happened to your smart meter?</summary>
  
![A sad smart meter](https://user-images.githubusercontent.com/5637/160574788-65638e0f-197f-4b0d-a2a0-5d13b8f7d613.png)

  1. `30 December 2021 00:00` Last meter reading received (visible at `api.octopus.energy/v1/electricity-meter-points`)
  2. `31 December 2021 13:57` Contacted Octopus
  3. `31 December 2021 18:21` Acknowledgement from Octopus
  4. `03 January 2022 14:54` Contacted Octopus again
  5. `03 January 2022 16:11` Octopus agrees to dispatch emergency meter engineer
  6. `03 January 2022 19:00` SMETS2 meter is installed but meter commissioning fails
  7. `03 January 2022 19:20` Contact Octopus to commission meter
  8. `16 February 2022 14:05` Contact Octopus (again) to commission meter
  9. `22 February 2022 09:43` Octopus requests information on meter status
  10. `22 February 2022 09:53` Contact Octopus with meter status
  11. `10 March 2022 18:41` Octopus agrees to dispatch meter engineer
  12. `24 March 2022 10:00` Communications hub replaced, meter commissioning fails
  13. `28 March 2022 14:00` Contact Octopus to understand next steps
</details>

It seems DCC tried [enrolling and adopting my SMETS1 meter](https://www.smartdcc.co.uk/our-smart-network/current-programmes/smets1-enrolment-and-adoption/) on 30 December 2021 but bricked the meter instead.

I've been told by Hildebrand my existing Glow Stick (actually given to me by Octopus) won't be useful for SMETS2 and even a new CAD won't be useful until my new meter can be commissioned. It's also likely my flat isn't within the coverage area for Arqiva's SMETS2 Scotland network.

I'm completely at a loss on how to move forward here :(

I'm hopeful Octopus will be able to figure a workaround with the DCC, but being unable to test my own meter has made it impossible to continue work on ha-hildebrandglow.

Apologies again for the inconvenience this has caused you all. I'm really enthused by the attention this project continues to receive and sincerely hope I can keep working on it in the near future.

**Comment by scott400k** [comment]
Don't suppose you know of a way to amend the current MQTT data to /10 as i'm getting far to higher readings (10* to much infact lol).
